tests/testflows/steps/cluster.py

```diff
@@ -144,8 +144,9 @@ def __enter__(self):
     def __exit__(self, type, value, traceback):
         try:
             with Finally("I clean up, docker compose cluster"):
-                # self.down()
-                pass
+                if settings.debug:
```

Comment:
do you mean?
```python
if no settings.debug():
  self.down()
```

Reply:
I have some problems with go code coverage, for now, I just check it. I will think how to do it properly